### ISSUE TITLE
Help Us Translate

### ISSUE THREAD

**Issue by elliotxx** [KusionStack/karpor#468]
TODO 🚧

**Comment by Paradiesvogel7** [comment]
i could help translating to german

**Comment by virtually-stray** [comment]
I would like to help translating it into Portuguese. How can I start? Should I make a FORK and translate the Read.me and link the translation on the original file?

**Comment by Paradiesvogel7** [comment]
> I would like to help translating it into Portuguese. How can I start? Should I make a FORK and translate the Read.me and link the translation on the original file?

i think thats the way to go. i found some instructions here:
https://github.com/KusionStack/community/blob/main/CONTRIBUTING.md#contribute-a-pull-request

**Comment by elliotxx** [comment]
Thanks @Paradiesvogel7 @virtually-stray ! 
Yes, you can translate in the FORK repo and then submit a PR to the upstream repo. More see the link of @Paradiesvogel7, you are right.

**Comment by elliotxx** [comment]
Add you to the contributors list, welcome! #472

**Comment by Paradiesvogel7** [comment]
> @Paradiesvogel7 Add you to the contributors list, welcome! #472

thx a lot

**Comment by virtually-stray** [comment]
I've started with only the README.md, just to get used to the workflow of the project. My plan is to translate the UI next, based on @Paradiesvogel7's changes, once I have a successful merge request.

If I've made any mistakes or if there are any corrections needed, please let me know.

**Comment by elliotxx** [comment]
@virtually-stray No problem, this is a good plan. I have already replied to you in the #474 .

**Comment by elliotxx** [comment]
@virtually-stray Add you to the contributors list, welcome! #475

**Comment by virtually-stray** [comment]
Thx man, appreciate it

**Comment by elliotxx** [comment]
@Paradiesvogel7 @virtually-stray Hi, I'd like to ask for advice, how do you know about karpor? we are curious because we haven't publicized it yet.

**Comment by Paradiesvogel7** [comment]
It's actually quite simple. I filtered for the good-first-issue tag on github and then randomly found the issue here.

**Comment by elliotxx** [comment]
> It's actually quite simple. I filtered for the good-first-issue tag on github and then randomly found the issue here.

Received, a simple and effective method 👍

**Comment by virtually-stray** [comment]
> It's actually quite simple. I filtered for the good-first-issue tag on github and then randomly found the issue here.

Yeah, similar situation here. I was looking for a good first project to contribute to and found this one in the goodfirstissues site. 

I also plan on keep contributing too, I've been just kinda busy this week, but I plan on contributing more soon. This project holds a lot of potential.

**Comment by elliotxx** [comment]
Thanks for recognition! There will be a lot of things to do for the new project. I welcome your joining and look forward to this project becoming a community-driven one.

**Comment by danielstankw** [comment]
I can do Polish

**Comment by elliotxx** [comment]
> I can do Polish

@danielstankw Coool~ Welcome to contribute! Do you need me to add more guide?

**Comment by Iamrushabhshahh** [comment]
Hey @elliotxx , 

Am intrested for helping on Hindi translation.

**Comment by elliotxx** [comment]
@Iamrushabhshahh  Very welcome! As mentioned in the issue, you can refer to the documentation to understand how to start contributing. If you have any questions, feel free to ping me
- https://github.com/KusionStack/community/blob/main/CONTRIBUTING.md#contribute-a-pull-request
- https://www.kusionstack.io/karpor/developer-guide/contribution-guide/

**Comment by Yash-Raj-5424** [comment]
Hi. I would love to add Korean language translation. 
Thanks !

**Comment by elliotxx** [comment]
@Yash-Raj-5424 Very welcome, thanks for the contribution!

**Comment by Prachi-Gupta2808** [comment]
hey, @elliotxx I would love to help with hindi translation, just give me a heads up if you guys are okay with it and then I can raise a PR. Since I am a beginner, would like to question that I just need to fork the repo and make read me file changes and raise a PR which you will review , right?

Thank you.
